IBM/CAST
# Csm rh8 build

# Purpose
this will help get csm building on rh8

# Origin
- https://github.ibm.com/CSM/team_process/issues/69
- https://github.ibm.com/CSM/team_process/issues/67
- https://github.ibm.com/CSM/team_process/issues/63

# How to Test
- we want to try to build another rpm after this has been pulled in.
- set up a csm env
- run an allocation
- basically the same test nick has already run on his own branch
- then confirming they all work as expected, give nate and IST the thumbs up to use

## eventual TODOs
- python not supported
- dcgm not supported
- but enough working to get started (ie you can make allocations with cgroups and such, run apis, install the database, etc etc)
